### ISSUE TITLE
Misplaced "Send Later" window

### ISSUE THREAD

**Issue by 121sim** [Extended-Thunder/send-later#147]
On MacOs 10.13.6, Thunberdird 78.5, when sending an email the window is sometimes misplaced : it's usually on the right of the screen (no problem then), but it's sometimes localised in the center of the screen.

On such cases, i can't confirm any option (send in 15, send in 30, etc.) because it's not clickable.

I just figure i can click on the right of the window (where it's usually localised), and it works.

To sum up, I have a misplaced and "dead" window, and a ghost but effective window aside.

Sorry for my english and my low-computer-level, and this firt GitHub post, and thanks for your work.  

Simon.

**Comment by jperryhouts** [comment]
Hm. I've seen similar bugs reported to Firefox and Thunderbird, but I can't seem to find links to any of those. Unfortunately this is beyond Send Later's control, because opening and rendering that popup is handled within Thunderbird itself.

How frequently does this happen? Does it happen more often when the window is in full screen mode, or some particular position on the screen?

Sorry I can't be more helpful, but there's very little I can do about this. You might consider opening a [Thunderbird bug report](https://bugzilla.mozilla.org/buglist.cgi?product=Thunderbird&component=Add-Ons%3A%20Extensions%20API&list_id=15500031) about it if it's a major problem for you.
